conans/requirements.txt

```
@@ -14,4 +14,5 @@ astroid>=1.6.5
 deprecation>=2.0, <2.1
 tqdm>=4.28.1, <5
 Jinja2>=2.3, <3
-python-dateutil>=2.6.0, <3
\ No newline at end of file
+python-dateutil>=2.6.0, <3
+typed-ast<1.4; python_version=="3.4" and platform_system=='Windows'
```

```suggestion
typed-ast<1.4; python_version == '3.4' and sys_platform == 'win32'
```

It works with `win32`, not with `windows`

---

platform_system and sys_platform are different, this PR seems to pass, I don't get your comment.

---

OMG!!  I didn't realize about the `platform_system` vs `sys_platform`